src/autoscaler/metricscollector/cmd/metricscollector/metricscollector_test.go

```diff
@@ -105,6 +105,8 @@ var _ = Describe("MetricsCollector", func() {
 
 			It("returns with a 200", func() {
 				rsp, err := http.Get(fmt.Sprintf("http://127.0.0.1:%d/v1/apps/an-app-id/metrics/memory", mcPort))
+				defer rsp.Body.Close()
```

Comment:
There are some other places that need to be fixed too.  Will include them in other  PR